app/routes.py

```diff
@@ -9,16 +9,27 @@
 #Home page
 @app.route('/', methods=['GET'])
 @app.route('/index', methods=['GET'])
-def index():
+@app.route('/index/<search_string>', methods=['GET'])
```

Comment:
This should be a query parameter - ?search=string. Please look at the Existing Code that we already have, as the implementation should be very similar. Using a / route is bad practice in this instance (cant search for create, login, submit, etc, and is inconsistent with what we are already doing.

Reply:
Done